test/IDE/complete_from_clang_framework.swift

```diff
@@ -28,8 +28,6 @@
 // RUN: %target-swift-ide-test -code-completion -source-filename %s -F %S/Inputs/mock-sdk -code-completion-token=CLANG_CLASS_MEMBERS_2 | %FileCheck %s -check-prefix=CLANG_CLASS_MEMBERS_2
 // RUN: %target-swift-ide-test -code-completion -source-filename %s -F %S/Inputs/mock-sdk -code-completion-token=CLANG_INSTANCE_MEMBERS_1 | %FileCheck %s -check-prefix=CLANG_INSTANCE_MEMBERS_1
 
-// XFAIL: linux
```

Comment:
EnableObjCInterop won't be on for these invocations, though, right?. They're all using -target and there's no argument for -enable-objc-interop in swift-ide-test.

Reply:
Hrm, I'm not super familiar with the code yet, but it looks to me like EnableObjCInterop is actually defaulted to `true` in `LangOptions.h`. A few of the tools default it to `isOSDarwin()` (`Frontend/CompilerInvocation.cpp`, `sil-llvm-gen`, `sil-opt`), but `swift-ide-test` doesn't modify the value. Not sure the best approach here, since `swift-ide-test` doesn't look like it plays much with flags currently.

Reply:
Interesting. I feel like swift-ide-test should follow the other tools to defaulting it based on the target triple and allowing that to be overridden.